.github/scripts/oss_build.functions.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -euo pipefail
+
+get_patch_part() {
```

Comment:
Can we add **function** directive for all functions to make it consistent ?

Reply:
Fixed https://github.com/hazelcast/hazelcast-docker/pull/740/commits/01a57c46f9172cd5a8b0facf4ae29fa214b752b3